src/client_side_reply.cc

```diff
@@ -1745,7 +1745,7 @@ clientReplyContext::doGetMoreData()
         localTempBuffer.data = getNextNode()->readBuffer.data;
         storeClientCopy(sc, http->storeEntry(), localTempBuffer, CacheHit, this);
     } else {
-        /* MISS CASE, http->logType is already set! */
+        /* MISS CASE, http->logType() is already set! */
```

Comment:
If you use `sed` to rename automatically, do not forget to manually fix the verb here:
```suggestion
        /* MISS CASE, http->loggingTags() are already set! */
```